src/components/atoms/AlertBox/AlertBox.tsx

```diff
@@ -21,9 +18,11 @@ const IconContainer = styled.div`
   margin-right: 16px;
 `;
 
-const AlertBox: FC<IAlertBoxProps> = ({ children, icon, ...rest }) => (
+const AlertBox: FC<HTMLAttributes<HTMLDivElement>> = ({ children, ...rest }) => (
```

Comment:
`<AlertBox />` is gonna be removed in favour of [`<Alert />`](https://zopauk.github.io/react-components/#/Components/Atoms/Alert) on a following PR, I just updated here to get **Typescript** happy 💆🏻‍♂️